compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/clauses/FunctionClauseNode.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.model.tree.clauses;
+
+import org.ballerinalang.model.tree.Node;
+import org.ballerinalang.model.tree.expressions.ExpressionNode;
+
+/**
+ * This interface represents the function clause in streams/tables SQLish syntax.
+ * <pre>Grammar:
+ *      FUNCTION functionInvocation
+ * </pre>
+ */
```

Comment:
Please add @Since 0.965.0 annotation for all the new classes

Reply:
added.

Reply:
added.